code/modules/lighting/lighting_object.dm

```diff
@@ -144,6 +144,9 @@
 /atom/movable/lighting_object/onTransitZ()
 	return
 
+/atom/movable/lighting_object/washed(var/washer)
```

Comment:
what the fresh hell is this and why doesn't the ABSTRACT flag control it

Reply:
because thats an item flag silly